<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1241</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Odswiez">
       <property name="text">
        <string>Odśwież</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_czas">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>25</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Zamknij">
       <property name="text">
        <string>Zamknij</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>901</width>
      <height>701</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTabWidget" name="Okna">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="AktuanaPogoda_2">
        <attribute name="title">
         <string>Aktualna Pogoda</string>
        </attribute>
        <widget class="QGraphicsView" name="Mapa_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>890</width>
           <height>670</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>730</x>
           <y>0</y>
           <width>161</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maxVisibleItems">
          <number>18</number>
         </property>
        </widget>
        <widget class="QPushButton" name="Gdansk">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>53</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Szczecin">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>162</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Olsztyn">
         <property name="geometry">
          <rect>
           <x>505</x>
           <y>105</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Bialystok">
         <property name="geometry">
          <rect>
           <x>690</x>
           <y>140</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Bydgoszcz">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>200</y>
           <width>51</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Gorzow">
         <property name="geometry">
          <rect>
           <x>107</x>
           <y>270</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Warszawa">
         <property name="geometry">
          <rect>
           <x>570</x>
           <y>275</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Wroclaw">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>430</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Torun">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>210</y>
           <width>31</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Zielona">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>340</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Opole">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>480</y>
           <width>51</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Krakow">
         <property name="geometry">
          <rect>
           <x>502</x>
           <y>530</y>
           <width>61</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Katowice">
         <property name="geometry">
          <rect>
           <x>450</x>
           <y>530</y>
           <width>41</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Rzeszow">
         <property name="geometry">
          <rect>
           <x>660</x>
           <y>530</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Lublin">
         <property name="geometry">
          <rect>
           <x>680</x>
           <y>370</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Lodz">
         <property name="geometry">
          <rect>
           <x>450</x>
           <y>340</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Poznan">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>270</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Kielce">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>440</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Historia_2">
        <attribute name="title">
         <string>Historia</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>190</y>
      <width>341</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonBia">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>BIAŁYSTOK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBYD">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>BYDGOSZCZ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonGDA">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>GDAŃSK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonGOR">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>GORZÓW WIELKOPOLSKI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonKAT">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>KATOWICE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonKIE">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>KIELCE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonKRA">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>KRAKÓW</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLUB">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>LUBLIN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLOD">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ŁÓDŹ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOLS">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OLSZTYN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOPO">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OPOLE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtoPOZ">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>POZNAŃ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRZE">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>RZESZÓW</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonsSZC">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SZCZECIN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonTOR">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TORUŃ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWAR">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>WARSZAWA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWRO">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>WROCŁAW</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonZIE">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ZIELONA GÓRA</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>130</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:16pt;&quot;&gt;LISTA MIAST DO WYBORU&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1252</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuaktualna_pogoda">
    <property name="title">
     <string>Aktualna pogoda w wybranym mieście</string>
    </property>
    <widget class="QMenu" name="menuMiasto">
     <property name="title">
      <string>Wybierz miasto</string>
     </property>
     <addaction name="actionBia_ystok"/>
     <addaction name="actionBydgoszcz"/>
     <addaction name="actionGda_sk"/>
     <addaction name="actionGorz_w_Wielkopolski"/>
     <addaction name="actionKatowice"/>
     <addaction name="actionKielce"/>
     <addaction name="actionKrak_w"/>
     <addaction name="actionLublin"/>
     <addaction name="action_d"/>
     <addaction name="actionOlsztyn"/>
     <addaction name="actionOpole"/>
     <addaction name="actionPozna"/>
     <addaction name="actionRzesz_w"/>
     <addaction name="actionSzczecin"/>
     <addaction name="actionToru"/>
     <addaction name="actionWarszawa"/>
     <addaction name="actionWroc_aw"/>
     <addaction name="actionZielona_G_ra"/>
    </widget>
    <addaction name="menuMiasto"/>
   </widget>
   <addaction name="menuaktualna_pogoda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBia_ystok">
   <property name="text">
    <string>Białystok</string>
   </property>
  </action>
  <action name="actionBydgoszcz">
   <property name="text">
    <string>Bydgoszcz</string>
   </property>
  </action>
  <action name="actionGda_sk">
   <property name="text">
    <string>Gdańsk</string>
   </property>
  </action>
  <action name="actionGorz_w_Wielkopolski">
   <property name="text">
    <string>Gorzów Wielkopolski</string>
   </property>
  </action>
  <action name="actionKatowice">
   <property name="text">
    <string>Katowice</string>
   </property>
  </action>
  <action name="actionKielce">
   <property name="text">
    <string>Kielce</string>
   </property>
  </action>
  <action name="actionKrak_w">
   <property name="text">
    <string>Kraków</string>
   </property>
  </action>
  <action name="actionLublin">
   <property name="text">
    <string>Lublin</string>
   </property>
  </action>
  <action name="action_d">
   <property name="text">
    <string>Łódź</string>
   </property>
  </action>
  <action name="actionOlsztyn">
   <property name="text">
    <string>Olsztyn</string>
   </property>
  </action>
  <action name="actionOpole">
   <property name="text">
    <string>Opole</string>
   </property>
  </action>
  <action name="actionPozna">
   <property name="text">
    <string>Poznań</string>
   </property>
  </action>
  <action name="actionRzesz_w">
   <property name="text">
    <string>Rzeszów</string>
   </property>
  </action>
  <action name="actionSzczecin">
   <property name="text">
    <string>Szczecin</string>
   </property>
  </action>
  <action name="actionToru">
   <property name="text">
    <string>Toruń</string>
   </property>
  </action>
  <action name="actionWarszawa">
   <property name="text">
    <string>Warszawa</string>
   </property>
  </action>
  <action name="actionWroc_aw">
   <property name="text">
    <string>Wrocław</string>
   </property>
  </action>
  <action name="actionZielona_G_ra">
   <property name="text">
    <string>Zielona Góra</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Zamknij</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1105</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>1060</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Odswiez</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
